### PR TITLE
Supply changes

### PR DESCRIPTION
**Pull request by Ruling-0**
- Railroad max level 5 >>> 7
- Railroad supply 10 + 5*level >>> 16 + 7*level
- State supply from VPs and infrastructure doubled
- Supply hub IC cost 6000 >>> 8000
- Naval base IC cost 3000 + 500*level >>> 4000 + 500*level

### CODE REVIEW

**Review by GrenadierSchubes — APPROVED** [review]
Good to go.

**Comment by iDealNimzi** [comment]
Can add me as a reviewer.  And then one more for approval to merge

**Comment by lolmemes369** [comment]
I still think railroads should go up to 10 to reach the same supply but other than that it seems fine

**Comment by Ruling-0** [comment]
> I still think railroads should go up to 10 to reach the same supply but other than that it seems fine

I agree with the sentiment, but the increased IC costs should prompt the increased investment while the changes can overall buff starting infrastructure

File: common/defines/00_defines.lua [hunk 1]
```diff
@@ -3416,7 +3416,7 @@ NCharacter = {
 },
 
 NSupply = {
-	MAX_RAILWAY_LEVEL = 5, -- update railway texture as well, each frame corresponds to a level
+	MAX_RAILWAY_LEVEL = 7, -- update railway texture as well, each frame corresponds to a level
```
Comment (iDealNimzi) [inline common/defines/00_defines.lua:3419]: Do we have to have a texture for higher than 5 railway level?

File: common/defines/00_defines.lua [hunk 2]
```diff
@@ -3482,10 +3482,10 @@ NSupply = {
 
 
 	-- used for calculating "flow" for railways.
-	RAILWAY_BASE_FLOW = 10.0, 		-- how much base flow railway gives when a node connected to its capital/a naval node by a railway
-	RAILWAY_FLOW_PER_LEVEL = 5.0, 	-- how much additional flow a railway level gives
-	RAILWAY_FLOW_PENALTY_PER_DAMAGED = 5.0, -- penalty to flow per damaged railway
-	RAILWAY_MIN_FLOW = 5.0, 		-- minimum railway flow can be reduced to
+	RAILWAY_BASE_FLOW = 16.0, 		-- how much base flow railway gives when a node connected to its capital/a naval node by a railway
+	RAILWAY_FLOW_PER_LEVEL = 7.0, 	-- how much additional flow a railway level gives
+	RAILWAY_FLOW_PENALTY_PER_DAMAGED = 7.0, -- penalty to flow per damaged railway
```
Comment (iDealNimzi) [inline common/defines/00_defines.lua:3487]: This is an interesting set of variables.  I wonder if there'd ever be a need to decrease the flow penalty a point while maintaining the flow_per_lvel.

**Review by iDealNimzi — APPROVED** [review]

**Comment by lolmemes369** [comment]
> 
infrastructure buff is fine. I'm more talking about supply capacity from the hub

**Review by Narizna — APPROVED** [review]

**Comment by GrenadierSchubes** [comment]
Some errors have been happening// Error y55 for supply it seems

**Pull request merged**